### PR TITLE
Android N

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AGDROID-570](https://issues.jboss.org/browse/AGDROID-570)

### CODE REVIEW

**Comment by danielpassos** [comment]
@matzew @secondsun 👀 ?

**Comment by secondsun** [comment]
+1, I tested it with several of the down stream projects and they seem to work just fine

**Comment by matzew** [comment]
+1

File: aerogear-android-bom/pom.xml [hunk 1]
```diff
@@ -30,9 +30,12 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <android.version>[5.0_r1,)</android.version>
-        <google.playservice.version>6.1.71</google.playservice.version>
-        <gson.version>2.2.2</gson.version>
+        <maven.compiler.target>1.8</maven.compiler.target>
```
Comment (danielpassos) [inline aerogear-android-bom/pom.xml]: Let's create a new Jira for it update it and move it to the _root_ pom

**Pull request merged**